### PR TITLE
add paddle.broadcast_to api which is a alias of paddle.expand

### PR DESCRIPTION
**Pull request by sandyhouse**
### PR types

New features
### PR changes

APIs
### Describe

add paddle.broadcast_to api which is a alias of paddle.expand

### CODE REVIEW

File: python/paddle/tensor/manipulation.py [hunk 1]
```diff
@@ -1223,6 +1223,96 @@ def expand_as(x, y, name=None):
...
+        N-D Tensor: A Tensor with the given shape. The data type is the same as ``x``.
+
+    Examples:
+        .. code-block:: python
+
+            import paddle
+
+            data = paddle.to_tensor([1, 2, 3], dtype='int32')
+            out = paddle.broadcast_to(data, shape=[2, 3])
+            out = out.numpy()
```
Comment (TCChenlong) [inline python/paddle/tensor/manipulation.py]: 这里print(out)就可以

File: python/paddle/tensor/manipulation.py [hunk 2]
```diff
@@ -1246,7 +1336,6 @@ def expand(x, shape, name=None):
 
             import paddle
 
-            paddle.disable_static()
             data = paddle.to_tensor([1, 2, 3], dtype='int32')
             out = paddle.expand(data, shape=[2, 3])
             out = out.numpy()
```
Comment (TCChenlong) [inline python/paddle/tensor/manipulation.py]: 同上

Comment (sandyhouse) [reply python/paddle/tensor/manipulation.py]: Done.

File: python/paddle/tensor/manipulation.py (hunk 1 above)
Comment (sandyhouse) [reply python/paddle/tensor/manipulation.py]: Done.

**Review by TCChenlong — APPROVED** [review]
LGTM

**Review by lanxianghit — APPROVED** [review]

**Pull request merged**